File: DAL/Models/Bank Information/Bank.cs
using DAL.Models.Customer_Related;
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models.Bank_Information
{
    public class Bank : AuditableEntity
    {
        public int Bank_ID { get; set; }
        public string Bank_Name { get; set; }
        //foreign key from bank types
        public int Bank_Type { get; set; }
        public ICollection<Bank_Account> Bank_Account { get; set; }
        public ICollection<Bank_Branch> Branch { get; set; }

        public Bank_Type bankType { get; set; }


    }
}

File: QuickApp/ClientApp/src/app/models/bank.ts
export class bank {

    constructor(Bank_Type_Name?:string,Bank_Type?:number,Bank_ID?: number, Bank_Name?: string) {

      
        this.Bank_ID = Bank_ID;
        this.Bank_Name = Bank_Name;
        this.Bank_Type_Name=Bank_Type_Name;
        this.Bank_Type=Bank_Type;
        
    }

    
    public Bank_ID: number;
    public Bank_Name: string;
    public Bank_Type_Name: string;
    public Bank_Type: number;
    
    

}